Install doesn't include hs_version.h

After compiling and installing vectorscan locally, the hs_version.h is missing in the installed header files:

```
/work/hyperscan-java-native/cppbuild/include/hs/hs.h:42:10: fatal error: hs_version.h: No such file or directory
   42 | #include "hs_version.h"
```

```
[root@e2de98379481 include]# find .
.
./hs
./hs/hs_common.h
./hs/hs_runtime.h
./hs/hs.h
./hs/hs_compile.h
```

It seems to me that this is related to the change making sure that the version numbers don't get out of sync:
https://github.com/VectorCamp/vectorscan/commit/16604f9539929b5d19dc81d529c36770fd8eebfd

Now hs_version.h is suddenly part of the required development headers, which was not the case before.

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -225,6 +225,7 @@ set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} ${EXTRA_CXX_FLAGS}")
 endif()
 
 SET(hs_HEADERS
+    ${PROJECT_BINARY_DIR}/hs_version.h
     src/hs.h
     src/hs_common.h
     src/hs_compile.h

